js/data/locale/tg/sysres.json

```diff
@@ -79,14 +79,14 @@
     "a1": "PM",
     "G-1": "BCE",
     "G1": "CE",
-    "1#1 millisecond|#{num} milliseconds": "#{num} ms",
-    "1#1 second|#{num} seconds": "#{num} s",
-    "1#1 minute|#{num} minutes": "#{num} min",
-    "1#1 hour|#{num} hours": "#{num} h",
-    "1#1 day|#{num} days": "#{num} d",
-    "1#1 week|#{num} weeks": "#{num} w",
-    "1#1 month|#{num} months": "#{num} m",
-    "1#1 year|#{num} years": "#{num} y",
+    "1#1 millisecond|#{num} milliseconds": "#{num} миллисекундҳо",
```

Comment:
I think using the first 3, 2 or 1 characters for the shorter forms is better than using the English letters, even if it is not completely correct in that language.